MvvmCross/Platform/Uap/Views/MvxBaseApplication.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Threading.Tasks;
+using MvvmCross.Platform.Uap.Core;
+using MvvmCross.Platform.Uap.Views.Suspension;
+using MvvmCross.ViewModels;
+using Windows.ApplicationModel;
+using Windows.ApplicationModel.Activation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+namespace MvvmCross.Platform.Uap.Views
+{
+    public abstract class MvxBaseApplication: Application
+    {
+        protected MvxWindowsSetup InternalSetup { get; set; }
```

Comment:
InternalSetup seems a bit strange as name. Especially since it is protected.

Reply:
Just "Setup" then?